usagov/usagov-2021
# USAGOV-1729-wizard-taxonomy-revisions: Enable wizard taxonomy revisions

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1729

## Description

Add drupal core patch in composer.json. Update wizard taxonomy configuration.

## Type of Changes

- [ X ] New Feature
  - Add screenshot showing what it should look like
- [ X ] Drupal Config (requires "drush cim")

## Testing Instructions


1. While logged in as a content editor, edit an existing wizard taxonomy term.
2. Make a change to one of the fields and save.
3. Open the revisions tab.
4. There is a new revision that was created with the change made in step 2.
5. Revert to the previous revision.
6. The revert functionality works as expected.
7. Ensure that you can still see the revisions tab when logged is as a content manager and content administrator.

### Change Requirements



## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
8. Find the commit of this pull request.
9. Build and deploy the changes.
10. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
